vmdb/app/models/ems_cluster.rb

```diff
@@ -317,7 +317,7 @@ def self.get_perf_collection_object_list
     end
 
     hids = cl_hash.collect do |a|
-      k, v = a
+      _, v = a
```

Comment:
Alternately you could use .values to eliminate the variable entirely

```ruby
hids = cl_hash.values.collect { |v| v[:ho_ids] }.flatten.compact.uniq
```

Oh wait, is that a flat_map I now see? ;)